src/components/Buttons/TransparentButton.tsx

```diff
@@ -0,0 +1,5 @@
+function TransparentButton({ onClick, ...rest }) {
```

Comment:
probably better to just extend `Button` component with e.g. optional prop `type` with value "transparent" or something like that? considering it differs by className only

Reply:
We are using `Button` component only as a transparent button in our app as of now.

I have refactored the `Button` component and can add more type to it in future. Thanks 🙏

Reply:
not sure i understand, i was proposing to remove TransparentButton, not sure it's worth adding separate component TransparentButton since it's only renders Button component with custom className?

what about having only this? 

```
components/Button
  ./Button.jsx
  ./Button.scss
```

or what is the reason to have "Common" and "Buttons" directories for buttons?